src/Jackalope/NodeType/NodeType.php

```diff
@@ -392,4 +393,28 @@ public function canRemoveProperty($propertyName, $throw = false)
 
         return true;
     }
+
+    /**
+     * @param $value
+     * @return array|mixed|string
+     */
+    private function getValueAsString($value)
+    {
+        if (is_object($value)) {
+            $val = get_class($value);
+        } elseif (is_scalar($value)) {
+            $val = (string)$value;
+        } elseif (is_array($value)) {
+            $val = [];
+            foreach ($value as $k => $v) {
+                if (is_object($v)) {
+                    $val[$k] = get_class($v);
```

Comment:
this will omit all values that are not objects. i think we should simplify this to `return 'array of length '.count($value);` instead of trying to show whats inside the array.